syft/pkg/cataloger/generic/cataloger.go

```diff
@@ -154,7 +154,7 @@ func (c *Cataloger) Catalog(ctx context.Context, resolver file.Resolver) ([]pkg.
 	var relationships []artifact.Relationship
 	var errs error
 
-	logger := log.Nested("cataloger", c.upstreamCataloger)
+	lgr := log.Nested("cataloger", c.upstreamCataloger)
```

Comment:
I assume this change is unrelated? Keeping a local variable name from shadowing a package name, but nothing to do with fixing the license scanner perf?

Reply:
unrelated to any performance gains. I was reviewing this code as a part of this work and removed some variable/package shadowing in the process (boy scouting).